tests/unit/test_row_data.py

```diff
@@ -385,10 +385,14 @@ def test_constructor(self):
         self.assertEqual(partial_rows_data.retry, DEFAULT_RETRY_READ_ROWS)
 
     def test_constructor_with_retry(self):
+        from google.cloud.bigtable.row_data import DEFAULT_RETRY_READ_ROWS
+
         client = _Client()
         client._data_stub = mock.MagicMock()
-        request = retry = object()
+        request = object()
+        retry = DEFAULT_RETRY_READ_ROWS
         partial_rows_data = self._make_one(client._data_stub.ReadRows, request, retry)
+        # self.assertEqual(partial_rows_data.response_iterator.timeout, 60.0)
```

Comment:
Can this line be uncommented now that the `api_core` feature is released?  Better yet, can we mock out the `read_method` and add an assertion that it was called with the expected timeout?

Reply:
@tseaver Absolutely, thanks for pointing this one out. PTAL.